### DISCUSSION TITLE
disable login for Dashboards

### DISCUSSION THREAD

**Discussion by Tsamsiyu1** [hassio-addons/addon-node-red#1188] [Q&A]
Is there a way to disable the login for dashboards? If I am correct this schould be the http-node. unfortunatly the integration config does not allow no password or deleting the option entiretly. I know you can disable it in the full node red install. Do i just miss something or is there maybe a way to send the password and username over the url to the dashboard.

**Comment by frenck** [comment]
https://github.com/hassio-addons/addon-node-red/blob/main/node-red/DOCS.md#option-leave_front_door_open

**Reply by Tsamsiyu1** [reply]
Failed to save add-on configuration, Missing required option 'testpassword' in Node-RED (a0d7b954_nodered). Got {'certfile': 'certificate.pem', 'credential_secret': 'testpassword', 'leave_front_door_open': True, 'http_node': {'password': None, 'username': None}, 'http_static': {'password': 'password', 'username': 'test'}, 'init_commands': [], 'keyfile': 'privkey.pem', 'npm_packages': [], 'ssl': True, 'system_packages': [], 'theme': 'default'}

**Reply by Tsamsiyu1** [reply]
am i missing something?

**Comment by Tsamsiyu1** [comment]
bumb
